src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="cards" data-qa="card">
+      <div class="cards__container">
+        <div class="cards__photo">
+          <img src="images/imac.jpeg" alt="imac" class="product-photo">
+        </div>
+        <div class="cards__name">
+          <div class="name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+        </div>
+
+        <div class="cards__kod kod">Код товара: 195434</div>
+        <div class="cards__rating">
+          <div class="rate">
+            <div class="stars stars--4">
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="cards_ratingComments">Отзывов: 5</div>
```

Comment:
Вынеси "cards_ratingComments" за блок "stars"
```suggestion
              <p class="cards_ratingComments">Отзывов: 5</p>
```